cumulus_library/study_parser.py

```diff
@@ -159,6 +161,15 @@ def clean_study(
         if not view_table_list:
             return view_table_list
 
+        # We'll do a pass to see if any of these tables were created outside of a
+        # study builder, and remove them from the list.
+        for view_table in view_table_list:
+            if any(
+                f"{self.get_study_prefix()}__{word}_" in view_table[0]
```

Comment:
nit: could be a `startswith` check, cause otherwise they could prepend stuff and stop using the prefix. But not important.

Reply:
that should be handled before this by prefix checking when the table is created - but i can make that change.